screens/settings/SettingsScreen.js

```diff
@@ -0,0 +1,146 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import { Updates } from 'expo';
+import Constants from 'expo-constants';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { AsyncStorage, Image, Linking, ScrollView, View } from 'react-native';
+import {
+  Body,
+  ButtonContainer,
+  NavButton,
+  NavHeaderContainer,
+  NavTitle,
+} from '../../components/BaseComponents';
+import CategoryBar from '../../components/resources/CategoryBar';
+import SettingsCard from '../../components/settings/SettingsCard';
+import Colors from '../../constants/Colors';
+import RecordIds from '../../constants/RecordIds';
+import { getCustomersById } from '../../lib/airtable/request';
+
+export default class SettingsScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isGuest: false,
+      name: '',
+      number: '',
+    };
+  }
+
+  // Load customer record & transactions
+  async componentDidMount() {
+    const customerId = await AsyncStorage.getItem('customerId');
+    const isGuest = customerId === RecordIds.guestCustomerId;
+    if (customerId != null && !isGuest) {
+      const customer = await getCustomersById(customerId);
+      const { name } = customer;
+      const number = customer.phoneNumber;
+      this.setState({ name, number });
+    }
+    this.setState({ isGuest });
+  }
+
+  _logout = async () => {
+    this.props.navigation.goBack();
+    await AsyncStorage.clear();
+    this.props.navigation.navigate('Auth');
+    Updates.reload();
+  };
+
+  render() {
+    return (
+      <View style={{ flex: 1 }}>
+        <NavHeaderContainer>
+          <NavButton onPress={() => this.props.navigation.goBack(null)}>
+            <FontAwesome5 name="arrow-left" solid size={24} />
+          </NavButton>
+          <NavTitle>Settings</NavTitle>
+        </NavHeaderContainer>
+        <ScrollView>
+          <CategoryBar icon="user" title="Account" />
+          {this.state.isGuest && (
+            <SettingsCard
+              title="Create an account"
+              description="Start earning Healthy Rewards"
+              navigation={this._logout}
```

Comment:
Not a huge thing but it would be smoother on the customer's end of the `Create an account` button took them immediately to the `SignUpScreen`?

Reply:
agreed!